RUFAS/output_manager.py

```diff
@@ -1003,10 +1002,10 @@ def flush_pools(self) -> None:
         """
         Sets each pool to an empty dictionary.
         """
-        self.variables_pool = {}
-        self.warnings_pool = {}
-        self.errors_pool = {}
-        self.logs_pool = {}
+        self.variables_pool: Dict[str, OutputManager.pool_element_type] = {}
+        self.warnings_pool: Dict[str, OutputManager.pool_element_type] = {}
+        self.errors_pool: Dict[str, OutputManager.pool_element_type] = {}
+        self.logs_pool: Dict[str, OutputManager.pool_element_type] = {}
```

Comment:
```suggestion
        self.variables_pool = {}
        self.warnings_pool = {}
        self.errors_pool = {}
        self.logs_pool = {}
```
I think I took out the type hint before, b/c this is a Mypy error of re-definition for the pools.

Reply:
Oh gotcha.